packages/lab/src/components/Flow/stories/Invalid/index.tsx

```diff
@@ -0,0 +1,96 @@
+import { useState } from "react";
+import { css } from "@emotion/css";
+import {
+  HvButton,
+  HvGlobalActions,
+  theme,
+} from "@hitachivantara/uikit-react-core";
+import { Backwards } from "@hitachivantara/uikit-react-icons";
+import { HvFlow } from "@hitachivantara/uikit-react-lab";
+import { ReactFlowInstance } from "reactflow";
+
+// The code for these values are available here: https://github.com/lumada-design/hv-uikit-react/tree/master/packages/lab/src/components/Flow/stories/Base/index.tsx
+import { nodeGroups, nodeTypes } from "../Base";
+import { ErrorBoundary } from "./ErrorBoundary";
+
+const invalidState = {
+  nodes: [
+    {
+      id: "b1831327e65",
+      // Remove the position to trigger the error
+      // position: { x: 230, y: -97 },
+      data: {},
+      type: "Asset",
+    },
+    {
+      id: "e654964ea39",
+      position: { x: 700, y: 88 },
+      data: {},
+      type: "mlModelDetection",
+    },
+  ],
+  edges: [
+    {
+      source: "b1831327e65",
+      sourceHandle: "1",
+      target: "e654964ea39",
+      targetHandle: "0",
+      id: "reactflow__edge-b1831327e651-e654964ea390",
+    },
+  ],
+  viewport: { x: 50, y: 300, zoom: 0.53 },
+};
+
+// Classes
+export const classes = {
+  root: css({ height: "100vh" }),
+  globalActions: css({ paddingBottom: theme.space.md }),
+  flow: css({
+    height: "calc(100% - 90px)",
+  }),
+};
+
+export const Invalid = () => {
+  const [instance, setInstance] = useState<ReactFlowInstance>();
+
+  const onInit = (inst: ReactFlowInstance) => {
+    setInstance(inst);
+  };
+
+  return (
+    <div className={classes.root}>
+      <HvGlobalActions
+        className={classes.globalActions}
+        position="relative"
+        backButton={
+          <HvButton aria-label="Back" icon>
+            <Backwards role="none" />
+          </HvButton>
+        }
+        title="New Flow"
+      >
+        <HvButton
+          variant="primary"
+          onClick={(e) => {
+            e.preventDefault();
+            // Ignore the error that comes with the invalid state
+            // @ts-ignore
+            instance?.setNodes(invalidState.nodes);
+            instance?.setEdges(invalidState.edges);
+          }}
+        >
+          Load Invalid Flow
+        </HvButton>
+      </HvGlobalActions>
+      <div className={classes.flow}>
+        <ErrorBoundary>
+          <HvFlow
+            nodeTypes={nodeTypes}
+            nodeGroups={nodeGroups}
+            onInit={onInit}
```

Comment:
I guess this is fine as is, but since the default state is an empty flow that we can only make invalid, we could start with the invalid state already, making the sample simpler

My "invalid button" suggestion was to turn a valid flow into an invalid one, ie.
```tsx
// @ts-expect-error invalidate flow by removing node positions
setNodes(nodes => nodes.map({position, ...rest} => rest));
```

Reply:
I'll update this.